deepaas/api/v2/train.py

```diff
@@ -62,6 +63,11 @@ def build_train_response(uuid, training):
                     ret["message"] = "%s" % exc
                 else:
                     ret["status"] = "done"
+                    ret["result"] = training["task"].result()
+                    ret["result"]["duration"] = str(datetime.datetime.strptime(ret["result"]["finish_date"],
+                                                                               '%Y-%m-%d %H:%M:%S.%f')
+                                                    - datetime.datetime.strptime(ret["date"],
+                                                                                 '%Y-%m-%d %H:%M:%S.%f'))
```

Comment:
Please change this to something like:
    end = ret["result"]...
    end = datetime....
    start = ....
    ret["result"]["duration] = str(end - start)